client/components/RatingsReviews/ReviewList.jsx

```diff
@@ -16,33 +20,48 @@ export const ReviewList = (props) => {
   // const [loading, setLoading] = useState(true);
   const [reviews, setReviews] = useState(<div>Loading reviews...</div>);
   const [moreReviews, setMoreReviews] = useState('');
-  const [initialized, setInitialized] = useState(false);
+  const [numReviews, setNumReviews] = useState(2);
+  const monthNames = ['January', 'February', 'March', 'April', 'May', 'June',
+    'July', 'August', 'September', 'October', 'November', 'December'
+  ];
   useEffect(() => {
+    let counter = 0;
     getReviews(props.id)
       .then(((data) => {
         setReviews(data.data.results.map((result) => {
-          return (
-            <div className='ReviewList' key={result.review_id}>
-              <div className="IndividualReview">
-                <div className="ReviewText">{result.summary}</div>
-                <div className="jStars">Stars Placeholder</div>
-                <div className="jDate">Date Placeholder</div>
-              </div>
-            </div>
-          );
+          counter++;
+          let date = new Date(result.date);
+          result.date = monthNames[date.getMonth()] + ' ' + date.getDate() + ', ' + date.getFullYear();
+          if (counter <= numReviews ) {
+            return (
+              <IndividualReview
+                result={result}
+                counter={counter}
+                date={date}
+                key={counter}
+              />
+            );
+          }
         }));
       }))
       .catch((err) => {
         throw err;
       });
-  }, []);
+  }, [numReviews]);
 
   return (
     <div className='ReviewList'>
       <div>{reviews}</div>
       <div className='jButtonContainer'>
-        <button className='jButton'>More Reviews</button>
-        <button className='jButton'>Add A Review</button>
+        { reviews.length > numReviews
```

Comment:
Instead of using a ternary operator, consider using the && operator, it is a common pattern when conditionally rendering JSX in react code.

(boolean expression && JSX) renders the JSX if and only if the boolean expression is true. Cleaner and more semantic